pkgs/applications/misc/nhentai/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, python3Packages }:
+python3Packages.buildPythonApplication rec {
```

```suggestion

python3Packages.buildPythonApplication rec {
```
This file could use some spacing, but otherwise LGTM.